src/views/Login.jsx

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import svg from "../assets/mobile_login.svg";
+import googleIcon from "../assets/google_icon.svg";
+import facebookIcon from "../assets/facebook_icon.svg";
+import barefootLogo from "../assets/barefoot_logo.svg";
+import Input from "../components/Input";
+
+function LoginForm(props) {
+  const googleAuth = () => {
+    window.open(
+      "https://bravo-bfn-be.herokuapp.com/api/v1/user/auth/google/",
```

Comment:
don't you think it make's more sense to use react hooks on this methods and also give some response to show the user login succeeded or failed either case

Reply:
that can't be implemented here because we receive a programmatic redirect from the backend